docs/lineage/manual-lineage-feature-guide.md

```diff
@@ -0,0 +1,123 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Manual Lineage
+
+<FeatureAvailability/>
+
+Starting in version `0.9.5`, DataHub supports the manual editing of lineage between entities in the UI. Add or remove upstream and downstream lineage edges in both the Lineage Visualization screen as well as the Lineage tab on entity pages. Use this feature to supplement automatic lineage extraction in ingestion or establish important entity relationships in sources that we don't support automatic extraction for yet! Manual lineage is only supported for Datasets, Charts, Dashboards, and Data Jobs for now.
+
+:::note
+
+If you edit lineage manually through the UI but have lineage automatically extracted in ingestion for the entities you edited, your edits from the UI may be overwritten when ingestion runs again.
+
+:::
+
+## Manual Lineage Setup, Prerequisites, and Permissions
+
+What you need to edit lineage in the UI:
+
+* **Edit Lineage** metadata privilege to edit lineage at the entity level
+
+You can create these privileges by creating a new [Metadata Policy](../authorization/policies.md).
+
+Keep in mind that you need the **Edit Lineage** metadata privilege on all entity types that you want to edit lineage for. So if you want to add a Chart upstream of a Dataset, you need the **Edit Lineage** privilege for both Charts and Datasets.
```

Comment:
"Keep in mind" makes me think we've already told them this. 

Let's say something more like:

"It is important to know that the Edit Lineage privilege is required for **all** entities whose lineage is affected by the changes. For example, in order to add "Dataset B" as an upstream dependency of "Dataset A", you'll need the **Edit Lineage** privilege for _both_ Dataset A and Dataset B."

Notice that I also use the Dataset -> Dataset case in the example, as this will most likely be the most common case for lineage editing

Reply:
yeah that makes sense to me!